### PR TITLE
Add metrics for JVM RCAs

### PR DESCRIPTION
**Pull request by rguo-aws**
*Issue #, if available:*

*Description of changes:*
Add metrics for JVM RCAs. Metrics will be emitted whenever an unhealthy RCA is triggered. 

*Tests:*
tested on docker

*Code coverage percentage for this patch:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/PerformanceAnalyzerApp.java [hunk 1]
```diff
@@ -77,6 +78,8 @@
       new SampleAggregator(RcaGraphMetrics.values());
   public static final SampleAggregator RCA_RUNTIME_METRICS_AGGREGATOR =
       new SampleAggregator(RcaRuntimeMetrics.values());
+  public static final SampleAggregator RCA_VERTICES_METRICS_AGGREGATOR =
```
Comment (ktkrg) [inline src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/PerformanceAnalyzerApp.java]: We should re-use the graph metrics aggregator. Thoughts?

File: src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/metrics/RcaVerticesMetrics.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.stats.eval.Statistics;
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.stats.measurements.MeasurementSet;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * metrics added by each RCA vertex in RCA graph. All metrics under this category are
+ * RCA specific
+ */
+public enum RcaVerticesMetrics implements MeasurementSet {
```
Comment (ktkrg) [inline src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/metrics/RcaVerticesMetrics.java]: Do we not want metrics around HotNode RCA?

File: src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/PerformanceAnalyzerApp.java (hunk 1 above)
Comment (rguo-aws) [reply src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/PerformanceAnalyzerApp.java]: yeah, maybe we should not create a separate aggregator. I was thinking we might add a lot more RCA metrics in the future which will make the  RCA_RUNTIME_METRICS_AGGREGATOR looks quite messy.

File: src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/metrics/RcaVerticesMetrics.java (hunk 2 above)
Comment (rguo-aws) [reply src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/metrics/RcaVerticesMetrics.java]: I will add that.

**Pull request merged**